src/easy_config/__init__.py

```diff
@@ -189,21 +190,13 @@ def load(
 
         :returns: an instance of the configuration class loaded with the parsed values
         """
-        values = {}
-        if _parse_files and cls.FILES:
-            for file_paths in cls.FILES:
-                values.update(cls._load_file(file_paths))
-        if _additional_files:
-            for files in _additional_files:
-                values.update(cls._load_file(files))
-        if _lookup_config_envvar is not None:
-            envvar = f'{cls.NAME.upper()}_{_lookup_config_envvar.upper()}'
-            file_name = os.environ.get(envvar)
-            if file_name:
-                values.update(cls._load_file(file_name))
-        if _parse_environment:
-            values.update(cls._load_environment())
-        values.update(cls._load_dict(kwargs))
+        values = ChainMap(*cls._load_helper(
+            _additional_files=_additional_files,
+            _parse_files=_parse_files,
+            _parse_environment=_parse_environment,
```

Comment:
@cthoyt it was broken because you missed passing this.

Reply:
LOL

Reply:
Right? Of course it was that.